x/liquidstakeibc/keeper/deposit.go

```diff
@@ -12,12 +12,12 @@ import (
 func (k *Keeper) SetDeposit(ctx sdk.Context, deposit *liquidstakeibctypes.Deposit) {
 	store := prefix.NewStore(ctx.KVStore(k.storeKey), liquidstakeibctypes.DepositKey)
 	bytes := k.cdc.MustMarshal(deposit)
-	store.Set([]byte(deposit.ChainId+deposit.Epoch.String()), bytes)
+	store.Set(append([]byte(deposit.ChainId), sdk.Uint64ToBigEndian(uint64(deposit.Epoch))...), bytes)
```

Comment:
Move this to `keys.go`

Reply:
used formatInt as you have used in other functions, and moved it